uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/UtilityNetworkTrace.qml

```diff
@@ -37,9 +37,7 @@ import QtQuick.Layouts
 
  The result Tab itself contains a TabBar of the results made up of custom Tabs defined in
  UtilityNetworkFunctionTraceResultsModel.
-
- Generic styles, displayed in this Qml UI, can be overwritten using the Calcite style.
-*/
+ /
```

Comment:
this is missing the asterisk to close out the comment

```suggestion
 */
```

Reply:
Good catch. Accepting the auto-commit change.